subgraphs/uniswap-v3/src/common/entities/account.ts

```diff
@@ -0,0 +1,33 @@
+import { Address } from "@graphprotocol/graph-ts";
+import { Account } from "../../../generated/schema";
+import { INT_ZERO } from "../constants";
+
+export function createAccount(address: Address): Account {
```

Comment:
I think this one is unused? Do you need to export it while having already `getOrCreateAccount`?

Reply:
No I don't suppose so XD. Removed